src/sensors/bno080sensor.cpp

```diff
@@ -175,8 +171,6 @@ void BNO080Sensor::motionLoop()
             imu.getAccel(v[0], v[1], v[2], acc);
             Network::sendAccel(v, PACKET_ACCEL);
         }
-        if (m_IntPin == 255 || imu.I2CTimedOut())
```

Comment:
My only concern is this change. It should be redundant because `BNO080::getReadings()` already checks whether INT pin is 255 and uses `waitForI2C()` to when receiving packet to check if I2C is timet out. But pls test.

Reply:
@Eirenliel please check if this works. (I don't have BNOs)

Reply:
I don't have anything, I live in an airbnb with a laptop 😢

Reply:
So FWIW, I tried this version with a BNO085 Tracker and it ran fine without any errors at like ~100 TPS when moving the tracker

Reply:
This check is specifically so it DOESN'T wait for I2C if there are no INT pin defined, otherwise it would go back to reading until it timed out, and can't do other stuff, like sending data or checking another tracker. So this is needed when int pin is not connected/not defined.

Reply:
ah okay, 
I just tried it with wrong INT Pins in defines.h (just set both of them to 0).
It resulted in constantly printing Sensor 0 does not respond error. 
An Aux MPU9250 was detected and working tho.

Reply:
INT set to 0 is still an INT, try with 255 - disabled.

Reply:
Or just return these two lines :D

Reply:
When INT is set to 255 it runs like normal. The BNO is detected and data is being read.

Reply:
It should get low TPS with aux if i understand correctly

Reply:
Jup, TPS is 100 but sometimes dropping to 60-70 for the BNO and the MPU

Reply:
I suspect that blocking manner of this loop could also be causing these TPS drops... @Eirenliel was it tested before with `if (imu.dataAvailable())` instead of `while (imu.dataAvailable())` by any chance?

Reply:
>  The first check it does, is whether INT pin is set to 255, and if not or pin level is LOW it receives the packet until I2C timeout.

That's the point, we don't want this to happen SECOND TIME. We just called dataAvailable(), we don't want to wait for i2c again. That's why it breaks if the INT pin is not used.

Just revert it please and it can be merged.